instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/server/SpringWebfluxTest.java

```diff
@@ -139,20 +140,22 @@ void basicGetTest(Parameter parameter) {
                   }
                   span.hasKind(SpanKind.INTERNAL)
                       .hasParent(trace.getSpan(0))
-                      .hasAttributesSatisfyingExactly(
-                          satisfies(
-                              CODE_FUNCTION,
-                              parameter.annotatedMethod == null
-                                  ? val -> val.isEqualTo("handle")
-                                  : val -> val.isEqualTo(parameter.annotatedMethod)),
-                          satisfies(
-                              CODE_NAMESPACE,
-                              parameter.annotatedMethod == null
-                                  ? val -> val.contains(INNER_HANDLER_FUNCTION_CLASS_TAG_PREFIX)
-                                  : val -> val.isEqualTo(TestController.class.getName())));
+                      .hasAttributesSatisfyingExactly(assertCodeFunction(parameter));
                 }));
   }
 
+  private static @NotNull AttributeAssertion assertCodeFunction(Parameter parameter) {
+    String expectedFunctionName =
+        parameter.annotatedMethod == null ? "handle" : parameter.annotatedMethod;
+    String expectedPrefix =
+        parameter.annotatedMethod == null
+            ? INNER_HANDLER_FUNCTION_CLASS_TAG_PREFIX
```

Comment:
`INNER_HANDLER_FUNCTION_CLASS_TAG_PREFIX` was previously used with a `contains` check. I guess `startsWith` like you use it is more accurate and still fine here?